CONTRIBUTING.md

```diff
@@ -122,6 +122,23 @@ To watch the logs of this IntelliJ instance, run `tail -f build/idea-sandbox/sys
 See [Robot plugin](https://github.com/JetBrains/intellij-ui-test-robot)
 and [a preso on testing UI of IntelliJ Plugins](https://slides.com/plipski/intellij-plugin-ui-testing) for more details.
 
+### macOS
+It is not possible to run `uiTest` gradle task in display mode `Xvfb` on macOS systems directly via `./gradlew`.
+You can run ui tests inside docker container (based on `gitmachete/intellij-plugin-ci:6.2.1` image) or using virtual machine.
+Sample configuration for launching docker container is shown below. Environment variables `UID` and `GID` placed into the container will be the user and group identifiers
+of the files created inside it.
+```shell
+docker run --rm -e UID=$UID -e GID=$GID \
+      -v [path to git-machete-intellij-plugin root directory e.g. "$PWD"]:/home/docker/git-machete-intellij-plugin \
```

Comment:
For even easier copying directly to the terminal, I'd leave:
```suggestion
      -v $PWD:/home/docker/git-machete-intellij-plugin \
```
and mention somewhere that it's assumed that current working directory is the top-level directory of the project